.github/CODEOWNERS

```diff
@@ -15,117 +15,96 @@
 
 ########## Build system ##########
 
-/Makefile*        @gares
-/dev/tools/make_git_revision.sh @gares
+/Makefile*                      @coq/legacy-build-maintainers
+/dev/tools/make_git_revision.sh @coq/legacy-build-maintainers
 
-/configure*       @ejgallego
+/configure*        @coq/legacy-build-maintainers @coq/build-maintainers
 
-/META.coq.in      @ejgallego
+/META.coq.in       @coq/legacy-build-maintainers
 
-/dev/build/windows @MSoegtropIMC
-# Secondary maintainer @maximedenes
+/dev/build/windows @coq/windows-build-maintainers
 
 ########## CI infrastructure ##########
 
-/dev/ci/          @coq/ci-maintainers
-/.travis.yml      @coq/ci-maintainers
-/.gitlab-ci.yml   @coq/ci-maintainers
-/Makefile.ci      @coq/ci-maintainers
-/dev/ci/nix	@coq/nix-maintainers
+/dev/ci/             @coq/ci-maintainers
+/.travis.yml         @coq/ci-maintainers
+/.gitlab-ci.yml      @coq/ci-maintainers
+/azure-pipelines.yml @coq/ci-maintainers
+/Makefile.ci         @coq/ci-maintainers
+
+/dev/ci/nix          @coq/nix-maintainers
+*.nix                @coq/nix-maintainers
 
 /dev/ci/user-overlays/*.sh @ghost
 # Trick to avoid getting review requests
 # each time someone adds an overlay
 
-/dev/ci/*.bat          @coq/ci-maintainers
-
-*.nix             @coq/nix-maintainers
-
-azure-pipelines.yml @coq/ci-maintainers
-/dev/ci/azure* @coq/ci-maintainers
-
 ########## Documentation ##########
 
-/README.md        @Zimmi48
-# Secondary maintainer @maximedenes
-
-/INSTALL*         @Zimmi48
-# Secondary maintainer @maximedenes
+/README.md             @coq/doc-maintainers
+/INSTALL.md            @coq/doc-maintainers
 
 /CODE_OF_CONDUCT.md    @Zimmi48
 # Secondary maintainer @mattam82
 
-/dev/doc/         @Zimmi48
-# Secondary maintainer @maximedenes
+/doc/                  @coq/doc-maintainers
+/Makefile.doc          @coq/doc-maintainers
+
+/dev/doc/              @coq/doc-maintainers
 
+/doc/changelog/*/*.rst @ghost
```

Comment:
Changelog updates do not trigger review request anymore as was requested during the WG.